sys/net/gnrc/routing/ipv6_auto_subnets/gnrc_ipv6_auto_subnets.c

```diff
@@ -316,6 +318,11 @@ static void _configure_subnets(uint8_t subnets, uint8_t start_idx, gnrc_netif_t
         } else {
             ext_opts = tmp;
         }
+
+        /* configure RPL root if applicable */
+        if (idx >= 0) {
+            gnrc_rpl_configure_root(downstream, &downstream->ipv6.addrs[idx]);
+        }
```

Comment:
Mhhh... I don't really like, how this accesses the _private_ address array here. How do you guarantee, that the address did not change in between adding the prefix and configuring the root?